packages/terra-toggle/docs/README.md

```diff
@@ -1,6 +1,6 @@
 # Terra Toggle
 
-Toggle component that transitions content in and out. For toggle button functionality, see [terra-toggle-button](https://engineering.cerner.com/terra-ui/#/components/terra-toggle-button/toggle-button/toggle-button).
+Toggle component that transitions content in and out. Use aria-expanded state on the triggering component. The objective of aria-expanded state is to indicate whether regions of the content are collapsible, and to expose whether a region is currently expanded or collapsed. For toggle button functionality, see [terra-toggle-button](https://engineering.cerner.com/terra-ui/#/components/terra-toggle-button/toggle-button/toggle-button).
```

Comment:
Let's add a note on using aria-controls as well like the example from https://www.w3.org/WAI/GL/wiki/Using_the_WAI-ARIA_aria-expanded_state_to_mark_expandable_and_collapsible_regions#Example_1:_Using_a_button_to_collapse_and_expand_a_region

Reply:
Resolved in 3478f79